nesc/whip6/apps/SCIFUARTPrintf/SCIFUARTPrintfPrv.nc

```diff
@@ -0,0 +1,26 @@
+/*
+ * whip6: Warsaw High-performance IPv6.
+ *
+ * Copyright (c) 2017 University of Warsaw
+ * All rights reserved.
+ *
+ * This file is distributed under the terms in the attached LICENSE
+ * files.
+ */
+#include <stdio.h>
+
+module SCIFUARTPrintfPrv {
+    uses interface Boot;
+    uses interface Timer<TMilli, uint32_t>;
+} implementation {
+    event void Boot.booted() {
+        call Timer.startWithTimeoutFromNow(2048);
```

Comment:
Is this 2048 ms ? It doesn't seem too demanding. Anyway, pls add comment with unit used here.